styles.css

```diff
@@ -241,14 +241,66 @@ nav {
 
 .grid-container {
   display: grid;
-  grid-template-columns: 1fr;
-  grid-gap: 1em;
-  height: 100%;
-  width: 100%;
-  /* grid-template-rows: repeat( auto-fill, minmax(375, 1fr) ); */
+  grid-template-columns: repeat(auto-fit, minmax(300px, 1fr));
+  grid-gap: 1em 0em;
+}
+
+@media screen and (min-width: 763px) {
```

Comment:
Adding comments inside of media queries will also help make your organization of items more clear.